target/linux/realtek/image/rtl930x.mk

```diff
@@ -31,6 +31,20 @@ define Device/xikestor_sks8300-8x
 endef
 TARGET_DEVICES += xikestor_sks8300-8x
 
+define Device/vimin_vm-s100-0800ms
+  SOC := rtl9303
+  UIMAGE_MAGIC := 0x93000000
+  DEVICE_VENDOR := Vimin
+  DEVICE_MODEL := VM-S100-0800MS
+  IMAGE_SIZE := 13312k
+  KERNEL_INITRAMFS := \
+	kernel-bin | \
+	append-dtb | \
+	gzip | \
+	uImage gzip
```

Comment:
Try to save space where possible. Please use lzma (bootloader should support it). And not only for initramfs but also for the normal image. If it does not work use rt-loader.